aptos-move/framework/aptos-framework/sources/block.move

```diff
@@ -163,7 +190,17 @@ module aptos_framework::block {
             failed_proposer_indices,
             time_microseconds: timestamp,
         };
-        emit_new_block_event(vm, &mut block_metadata_ref.new_block_events, new_block_event);
+        let new_block_event_v2 = NewBlock {
+            hash,
+            epoch,
+            round,
+            height: block_metadata_ref.height,
+            previous_block_votes_bitvec,
+            proposer,
+            failed_proposer_indices,
+            time_microseconds: timestamp,
+        };
+        emit_new_block_event(vm, &mut block_metadata_ref.new_block_events, new_block_event, new_block_event_v2);
```

Comment:
if we're doing this, we might incur a performance impact on each block before we even migrate.

Reply:
one more event per block? should not be an issue regarding perf.